docs/topics/db/models.txt

```diff
@@ -964,6 +964,35 @@ abstract base class. For example, including ``db_table`` would mean that all
 the child classes (the ones that don't specify their own :ref:`Meta <meta-options>`) would use
 the same database table, which is almost certainly not what you want.
 
+If a child class inherits from multiple abstract base classes, only the
```

Comment:
I feel like an introductory clause that this is due to Python's inheritance rules could call out that this should be expected by users familiar with Python classes:

```
Due to Python's inheritance rules, ...
```

or similra

Reply:
Thanks Adam - that's a good addition.